### ISSUE TITLE
Sort imports not working

### ISSUE THREAD

**Issue by GiscardBiamby** [charliermarsh/ruff-vscode#152]
I'm running on vscode, i've tried both local on MacBook and remote-ssh session on a linux machine. 

Whenever I try to use ruff to organize imports nothing happens to the imports. I've purposely put the imports out of alphabetical order to test.

I've also tried installing ruff into my python environment and specifying the ruff path from the vscode settings, as well as specifying the python interpreter path for use by ruff.

If I switch back to using isort it works fine. I tried uninstalling isort from both my python environment as well as the ms isort extension. 

None of the things I tried above made any difference. However, ruff seems to be working great and extremely fast for lining and auto-fixing code issues! Very nice tool so far, even if I have to fallback to isort for import sorting.

```bash
[Trace - 12:32:53 AM] Sending request 'workspace/executeCommand - (55)'.
Params: {
    "command": "ruff.applyOrganizeImports",
    "arguments": [
        {
            "uri": "file:///home/gbiamby/proj/semafor/components/tonic-berkeley-template/scripts/run_tests.py",
            "version": 38
        }
    ]
}


[Trace - 12:32:53 AM] Received notification 'window/logMessage'.
Params: {
    "type": 4,
    "message": "Using interpreter executable: /home/gbiamby/mambaforge/envs/sem_env/bin/ruff"
}


Using interpreter executable: /home/gbiamby/mambaforge/envs/sem_env/bin/ruff
[Trace - 12:32:53 AM] Received notification 'window/logMessage'.
Params: {
    "type": 4,
    "message": "Running Ruff with: ['/home/gbiamby/mambaforge/envs/sem_env/bin/ruff', '--force-exclude', '--no-cache', '--no-fix', '--quiet', '--format', 'json', '-', '--fix', '--extend-ignore', 'ALL', '--extend-select', 'I001', '--stdin-filename', '/home/gbiamby/proj/semafor/components/tonic-berkeley-template/scripts/run_tests.py']"
}


Running Ruff with: ['/home/gbiamby/mambaforge/envs/sem_env/bin/ruff', '--force-exclude', '--no-cache', '--no-fix', '--quiet', '--format', 'json', '-', '--fix', '--extend-ignore', 'ALL', '--extend-select', 'I001', '--stdin-filename', '/home/gbiamby/proj/semafor/components/tonic-berkeley-template/scripts/run_tests.py']
[Trace - 12:32:53 AM] Received request 'workspace/applyEdit - (28096ebc-c7b1-4007-9ee2-9130a1f8ea1d)'.
Params: {
    "edit": {
        "documentChanges": [
            {
                "textDocument": {
                    "uri": "file:///home/gbiamby/proj/semafor/components/tonic-berkeley-template/scripts/run_tests.py",
                    "version": 38
                },
                "edits": []
            }
        ]
    },
    "label": "Ruff: Format imports"
}


[Trace - 12:32:53 AM] Received response 'workspace/executeCommand - (55)' in 9ms.
No result returned.


[Trace - 12:32:53 AM] Sending response 'workspace/applyEdit - (28096ebc-c7b1-4007-9ee2-9130a1f8ea1d)'. Processing request took 8ms
Result: {
    "applied": false
}


[Trace - 12:32:58 AM] Sending notification 'textDocument/didSave'.
Params: {
    "textDocument": {
        "uri": "file:///home/gbiamby/proj/semafor/components/tonic-berkeley-template/scripts/run_tests.py"
    }
}


[Trace - 12:32:58 AM] Received notification 'window/logMessage'.
Params: {
    "type": 4,
    "message": "Using interpreter executable: /home/gbiamby/mambaforge/envs/sem_env/bin/ruff"
}


Using interpreter executable: /home/gbiamby/mambaforge/envs/sem_env/bin/ruff
[Trace - 12:32:58 AM] Received notification 'window/logMessage'.
Params: {
    "type": 4,
    "message": "Running Ruff with: ['/home/gbiamby/mambaforge/envs/sem_env/bin/ruff', '--force-exclude', '--no-cache', '--no-fix', '--quiet', '--format', 'json', '-', '--stdin-filename', '/home/gbiamby/proj/semafor/components/tonic-berkeley-template/scripts/run_tests.py']"
}


Running Ruff with: ['/home/gbiamby/mambaforge/envs/sem_env/bin/ruff', '--force-exclude', '--no-cache', '--no-fix', '--quiet', '--format', 'json', '-', '--stdin-filename', '/home/gbiamby/proj/semafor/components/tonic-berkeley-template/scripts/run_tests.py']
[Trace - 12:32:58 AM] Received notification 'textDocument/publishDiagnostics'.
Params: {
    "uri": "file:///home/gbiamby/proj/semafor/components/tonic-berkeley-template/scripts/run_tests.py",
    "diagnostics": [
        {
            "range": {
                "start": {
                    "line": 5,
                    "character": 100
                },
                "end": {
                    "line": 5,
                    "character": 128
                }
            },
            "message": "Line too long (128 > 100 characters)",
            "severity": 2,
            "code": "E501",
            "source": "Ruff",
            "data": {
                "fix": null,
                "noqa_row": 8
            }
        },
        {
            "range": {
                "start": {
                    "line": 6,
                    "character": 100
                },
                "end": {
                    "line": 6,
                    "character": 119
                }
            },
            "message": "Line too long (119 > 100 characters)",
            "severity": 2,
            "code": "E501",
            "source": "Ruff",
            "data": {
                "fix": null,
                "noqa_row": 8
            }
        },
        {
            "range": {
                "start": {
                    "line": 362,
                    "character": 100
                },
                "end": {
                    "line": 362,
                    "character": 128
                }
            },
            "message": "Line too long (128 > 100 characters)",
            "severity": 2,
            "code": "E501",
            "source": "Ruff",
            "data": {
                "fix": null,
                "noqa_row": 363
            }
        }
    ]
}


[Trace - 12:32:58 AM] Sending request 'textDocument/codeAction - (56)'.
Params: {
    "textDocument": {
        "uri": "file:///home/gbiamby/proj/semafor/components/tonic-berkeley-template/scripts/run_tests.py"
    },
    "range": {
        "start": {
            "line": 33,
            "character": 57
        },
        "end": {
            "line": 33,
            "character": 57
        }
    },
    "context": {
        "diagnostics": []
    }
}


[Trace - 12:32:58 AM] Received response 'textDocument/codeAction - (56)' in 3ms.
Result: [
    {
        "title": "Ruff: Fix All",
        "kind": "source.fixAll",
        "diagnostics": [],
        "data": "file:///home/gbiamby/proj/semafor/components/tonic-berkeley-template/scripts/run_tests.py"
    }
]


[Trace - 12:32:58 AM] Sending request 'textDocument/codeAction - (57)'.
Params: {
    "textDocument": {
        "uri": "file:///home/gbiamby/proj/semafor/components/tonic-berkeley-template/scripts/run_tests.py"
    },
    "range": {
        "start": {
            "line": 33,
            "character": 57
        },
        "end": {
            "line": 33,
            "character": 57
        }
    },
    "context": {
        "diagnostics": []
    }
}


[Trace - 12:32:59 AM] Received response 'textDocument/codeAction - (57)' in 1ms.
Result: [
    {
        "title": "Ruff: Fix All",
        "kind": "source.fixAll",
        "diagnostics": [],
        "data": "file:///home/gbiamby/proj/semafor/components/tonic-berkeley-template/scripts/run_tests.py"
    }
]



```

**Comment by charliermarsh** [comment]
Are other autofix actions working? E.g., removing unused imports?

**Comment by vasudev-gm** [comment]
@charliermarsh I think we can merge this issue [https://github.com/charliermarsh/ruff-vscode/issues/151](url) because Sort Imports didn't work after 1.76.1 on windows and linux. Now @GiscardBiamby is seeing same issue on a mac. It seems Vscode team has released an updated version 1.76.2.

**Comment by charliermarsh** [comment]
Ah ok, merging into https://github.com/charliermarsh/ruff-vscode/issues/151.
